src/App.tsx

```diff
@@ -1,73 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Dropdown } from './components/Dropdown/Dropdown';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const { name, born, died } = selectedPerson || peopleFromServer[0];
```

Comment:
```suggestion
  const { name, born, died } = selectedPerson || {};
```
it could work as well
explicitly have `name, born, died` undefined if there is no `selectedPerson`